### ISSUE TITLE
In model.fit() add verbose=3 option that allows to see loss and accuracy per batch.

### ISSUE THREAD

**Issue by lugi7** [keras-team/keras#8177]
Having model.fit print the mean error for all the previous batches (same for accuracy) is just not useful at all. Verbose=1 is used mainly when prototyping, for example making sure loss doesn't explode in the middle of an epoch. Therefore there should be an option to print the loss and accuracy only for current batch, as it gives much more information that is useful.
